src/Uno.UI/UI/Xaml/Media/Imaging/RenderTargetBitmap.skia.cs

```diff
@@ -0,0 +1,58 @@
+﻿#nullable enable
+using System.Runtime.InteropServices;
+using Windows.Foundation;
+namespace Windows.UI.Xaml.Media.Imaging
+{
+	partial class RenderTargetBitmap
+	{
+		delegate void SwapColor(ref byte[] buffer, int byteCount);
+
+		static readonly SwapColor? PlatformSwap = RuntimeInformation.IsOSPlatform(OSPlatform.OSX)
+			? new SwapColor(SwapRB)
+			: default(SwapColor);
+
+		private static (int ByteCount, int Width, int Height) RenderAsBgra8_Premul(UIElement? element, ref byte[]? buffer, Size? scaledSize = null)
+		{
+			UIElement elementToRender = element
+				?? XamlRoot.Current.Content
+				?? throw new global::System.NullReferenceException();
```

Comment:
It's generally best to avoid raising `NullReferenceException`. Use `InvalidOperationException` instead, with a message.